__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -363,6 +390,7 @@ describe('TESTING ChatSchema', () => {
         isMe: true,
         message: 'message',
         rawJson: expect.any(String),
+        attachmentJson: expect.any(String),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Here, `attachmentJson` is expected to be a string. This is correct if we are storing JSON strings in the database.



<!-- This is an auto-generated comment by OSS CodeRabbit -->